tests/buf_writer.rs

```diff
@@ -47,6 +47,7 @@ fn test_buffered_writer() {
     })
 }
 
+#[ignore]
```

Comment:
I'm curious, what's the error here?

Reply:
This assertion is failed:
https://github.com/async-rs/async-std/blob/ca80ca981ea55374a5b1d29819fe3d2e337dd916/tests/buf_writer.rs#L57

Error:
```text
---- test_buffered_writer_inner_into_inner_does_not_flush stdout ----
thread 'test_buffered_writer_inner_into_inner_does_not_flush' panicked at 'assertion failed: `(left == right)`
  left: `[0, 1]`,
 right: `[]`', tests/buf_writer.rs:58:9
```

I think this is because #315 changed the behavior of lnto_inner: https://github.com/async-rs/async-std/pull/315/files#diff-a2b217372e599101d2b0f1dbab63ee2eR205